config/routes.rb

```diff
@@ -12,6 +12,8 @@
   match '/auth/:provider/callback', to: 'sessions#create',  via: [:get, :post]
   match '/auth/failure',            to: 'sessions#failure', via: [:get, :post]
 
+  match "/404", :to => "errors#error404", :via => :all, as: :not_found
```

Comment:
I'd like to see this look like `match "/404", to: 'errors#not_found`, via: :all, as: :not_found.